### PR TITLE
Resolve monthly fee bug + improve test coverage

### PR DESCRIPTION
**Pull request by enhao25**
Resolve a bug where monthly fee cannot be 0, so when we run `clear` the app, the fee doesn't work.

### CODE REVIEW

**Review by chwoozy — APPROVED** [review]
LGTM! 👍🏼

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 1]
```diff
@@ -93,6 +95,17 @@ public void parseStudentCommand_list() throws Exception {
         assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD + " 3") instanceof ListCommand);
     }
 
+    @Test
+    public void parseEmailCommand() throws Exception {
+        assertTrue(parser.parseCommand(EmailCommand.COMMAND_WORD) instanceof EmailCommand);
+    }
```
Comment (samleewy) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:101]: thanks for helping!

File: src/test/java/seedu/address/model/student/AddressTest.java [hunk 2]
```diff
@@ -19,6 +20,12 @@ public void constructor_invalidAddress_throwsIllegalArgumentException() {
         assertThrows(IllegalArgumentException.class, () -> new Address(invalidAddress));
     }
 
+    @Test
+    public void hashcode_equals() {
+        String address = "Blk 123 Outer space road";
+        assertEquals(new Address(address).hashCode(), new Address(address).hashCode());
```
Comment (samleewy) [inline src/test/java/seedu/address/model/student/AddressTest.java:26]: does this and the remaining hashcode test cases somehow increase coverage? hahaha

**Review by samleewy — APPROVED** [review]
looks good! thanks!

**Pull request merged**